client/my-sites/checkout/src/components/checkout-main.tsx

```diff
@@ -545,6 +548,7 @@ export default function CheckoutMain( {
 			isLoading: responseCart.products.length < 1,
 		},
 		{ name: translate( 'Loading countries list' ), isLoading: countriesList.length < 1 },
+		{ name: translate( 'Loading Site' ), isLoading: isCollapseExperimentLoading },
```

Comment:
Is 'site' typically capitalized?

Reply:
I just used this string because it already exists so we wouldn't need to translate a new one. A more accurate string would be "Loading experiment" but then it would need to be translated which would delay this work. And it isn't really useful anyway. This will only be shown if loading the experiment is very slow and is essentially for debugging. If it's reported to us, we will know that "Loading Site" means "Loading experiment".